core/templates/dev/head/css/oppia.css

```diff
@@ -2604,7 +2604,7 @@ div#ng-curtain {
 }
 
 .oppia-navbar-brand-name {
-  float: left;
+  float: right;
```

Comment:
Just to check -- on desktop this still looks good right?

Reply:
Yes, it does.

<img width="1678" alt="Screenshot 2019-11-20 at 2 11 32 AM" src="https://user-images.githubusercontent.com/15226041/69184601-2fbb4700-0b3b-11ea-9f7c-f952d634d1ee.png">